src/duties/autoFix.js

```diff
@@ -11,12 +11,16 @@ let autoFix = async ({ message }) => {
     let Repo = await Repository.open(repoDir)
     let statuses = await Repo.getStatus()
 
-    let modifiedFiles = statuses.map(file => !!file.isModified())
-    let hasModifiedFiles = _.contains(true, modifiedFiles)
-
-    if (hasModifiedFiles) {
+    let modifiedFiles = statuses.filter(file => {
+      let isJsFile = /.*.js$/.test(file.path())
```

Comment:
We can't just filter js files because some tools might do things like format sass files. We can make this part of the per-project configuration, or we can try to figure out why there are undesirable files being changed as part of the ci process.